### PR TITLE
ref: Add more codeowners

### PR DESCRIPTION
**Pull request by untitaker**
Pierre should not be blocked on changes to the schema, he sufficiently
owns both ends of the pipe

### CODE REVIEW

**Comment by phacops** [comment]
I would love more access like this. I would also include:
- `metrics_summaries` (`@getsentry/performance`)
- `spans` (`@getsentry/performance`)
- `profiles` (`@getsentry/profiling`)
- `functions` (`@getsentry/profiling`)

File: CODEOWNERS [hunk 1]
```diff
@@ -15,7 +15,7 @@
 /topics/processed-profiles.yaml                            @getsentry/owners-snuba @getsentry/profiling
 /topics/profiles-call-tree.yaml                            @getsentry/owners-snuba @getsentry/profiling
 /topics/group-attributes.yaml                              @getsentry/owners-snuba @getsentry/issues
-/topics/snuba-spans.yaml                                   @getsentry/owners-snuba
+/topics/snuba-spans.yaml                                   @getsentry/profiling
```
Comment (lynnagara) [inline CODEOWNERS:18]:
```suggestion
/topics/snuba-spans.yaml                                   @getsentry/owners-snuba @getsentry/profiling
```

**Review by lynnagara — CHANGES_REQUESTED** [review]
owners-snuba should still be on this like all other snuba topics

**Comment by untitaker** [comment]
it's not clear to me why we have codeowners for the topic yaml inconsistent with the examples and schema file. IMO those all should be the same

**Review by lynnagara — APPROVED** [review]

**Pull request merged**